File: backend/src/main/java/org/example/backend/Entity/Team.java
package org.example.backend.Entity;

import jakarta.persistence.*;
import lombok.Data;
import org.hibernate.annotations.CreationTimestamp;

import java.util.Date;

@Entity
@Data
@Table(name = "team")
public class Team {

    @Id
    @Column(name = "id")
    private long id;

    @Column(name = "steps_goal")
    private long stepsGoal;

    @CreationTimestamp
    @Column(name = "created_at")
    private Date createdAt;

}

File: frontend/src/models/Team.ts
export class Team {
    id: number
    stepsGoal: number
    createdAt: Date

    constructor(id: number, stepsGoal: number, createdAt: Date) {
        this.id = id
        this.stepsGoal = stepsGoal
        this.createdAt = createdAt
    }
}